test/integration_test_helper.rb

```diff
@@ -20,18 +20,49 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
-Capybara.register_driver :selenium_chrome do |app|
+
+javascript_driver = ENV.fetch("JS_TEST_DRIVER") { ENV['DEBUG_JS_TEST'] ? :selenium_chrome : :selenium_chrome_headless }.to_sym
+
+def get_chrome_options
   options = Selenium::WebDriver::Chrome::Options.new
-  options.args << '--disable-gpu'
-  options.args << '--no-sandbox'
-  options.args << '--window-size=1024,768'
-  options.args << '--headless' unless ENV['DEBUG_JS_TEST'] == '1'
-  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+  additional_options = ENV.fetch('ADDITIONAL_CHROME_OPTIONS', '')
+  additional_options.split(";").each do |add_opt|
+    options.args << add_opt
+  end
```

Comment:
Isn't this equivalent?
```suggestion
  options.args += ENV.fetch('ADDITIONAL_CHROME_OPTIONS', '').split(';')
```